content/en/apps/tutorials/application-tests.md

```diff
@@ -50,18 +56,30 @@ Writing tests for CHT apps requires a good understanding of the project workflow
     ```
     After adding these scripts, you are able to run the tests by running one of these commands from the command-line:
 
-   |Command|Description|
-   |---|---|
-   |`npm run test`|Compiles the app settings, converts the forms, then runs the tests (preferred solution).
-   |`npm run unittest`|Only runs the tests.|
+    |Command|Description|
+    |---|---|
+    |`npm run test` |Compiles the app settings, converts the forms,then runs the tests (preferred solution).|
+    |`npm run unittest`| Only runs the unit tests.|
 
-4. Create a folder in the project root where you keep the tests. You can name the folder yourself. For convenience, here it is named as `test`.
-5. Create a file `harness.defaults.json` at the root of your project. This is the default configuration file for the harness. Here you can specify the default user, preloaded contacts and reports, and add other settings. Here's an [example](https://github.com/medic/cht-conf-test-harness/blob/master/harness.defaults.json.example) file to get you started. You can read more about it [here](https://docs.communityhealthtoolkit.org/cht-conf-test-harness/global.html#HarnessInputs).
 
+     You can also run the test on a specific form by defining a test under the scripts section.
+     
+     For example, to run the test on the `death.spec.js` file, define the test as `death-form-unit-test: "mocha test/forms/death.spec.js --timeout 20000"`. Then run:
+     ```shell
+        npm run death-form-unit-test
+     ```   
+   
+     Npm also supports a shorter alias as `npm test`.There are recommended conventions on how to arrange tests by splitting into unit tests and integration tests so that each of them can be run independently. This can be best illustrated by [this project code](https://github.com/medic/cht-pih-malawi-app).
```

Comment:
It does not look right to write `npm` as `Npm`. Also, space needed after period.
```suggestion
     npm also supports a shorter alias as `npm test`. There are recommended conventions on how to arrange tests by splitting into unit tests and integration tests so that each of them can be run independently. This can be best illustrated by [this project code](https://github.com/medic/cht-pih-malawi-app).
```